src/windows/SliderWindows/SliderView.cpp

```diff
@@ -237,4 +235,13 @@ namespace winrt::SliderWindows::implementation {
                 });
         }
     }
+
+    const double SliderView::CalculateStepFrequencyPercentageValue(const double& stepPropertyValue) const noexcept {
```

Comment:
there's no point in passing a primitive type by reference if you don't modify the incoming parameter

Reply:
While marking the function as an accessor with the latter 'const', I don't think that making the return type 'const double' changes anything